src/otx/v2/adapters/torch/lightning/modules/datasets/dataset.py

```diff
@@ -0,0 +1,370 @@
+"""Dataset & DataModule for OTX lightning modules."""
```

Comment:
How about we rename it to something like `visual_prompting.py`? This way, we can have other `lightning->otx` datasets in the same folder.

Reply:
Thanks for the comment.
I renamed this to `visual_prompting_dataset.py`.